lib/trento_web/openapi/schema/runner.ex

```diff
@@ -0,0 +1,114 @@
+defmodule TrentoWeb.OpenApi.Schema.Runner do
+  @moduledoc false
+
+  require OpenApiSpex
+  alias OpenApiSpex.Schema
+
+  defmodule ResourceIdentifier do
```

Comment:
I guess we are already visualizing and environment where we can run checks beyond only clusters hehe

Otherwise, I would call this `ClusterIdentifier`.
But I find it totally fine as it is